languages/rusin/apk/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="rusin_app_name">Rusin for AnySoftKeyboard</string>
```

Comment:
This should be in the native language.

Reply:
It is fixed to be Rusyn for AnySoftKeyboard.
I see other languages like Russian and Belarus did not used cyrilic letters too so.